curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/6553efd6ada3f42aa2d75448.md

```diff
@@ -16,10 +16,18 @@ def function_name(param_1, param_2):
 
 Parameters are variables that you can use inside your function. A function can be declared with different number of parameters. In the example above, `param_1` and `param_2` are parameters.
 
-Modify your function declaration so that it takes two parameters called `message` and `offset`.
+Modify your function declaration so that it takes two parameters called message and offset. After that, you'll see an error appear in the terminal.
```

Comment:
```suggestion
Modify your function declaration so that it takes two parameters called message and offset.

After that, you'll see an error appear in the terminal. As a temporary fix, comment out the `caesar()` call.
```

Reply:
I cannot leave the suggestion, but we need to expand the editable region to allow to comment out the function call.